utilities/private/ft_preamble_init.m

```diff
@@ -46,6 +46,20 @@
   error(msg);
 end % if nargin
 
+% check if there are fieldnames in the cfg that suggest as if the user
+% erroneously inputted a data argument
+checkdatafields = isfield(cfg, {'label' 'dimord' 'time' 'trialinfo' 'avg'});
+if any(checkdatafields)
+  stack = dbstack('-completenames');
+  stack = stack(3);
+  help(stack.name);
+  % throw the error as if it happened in the original function
```

Comment:
this is interesting, this is also how we should do it in ft_notification when called by ft_error.